scripts/workers/submission_worker.py

```diff
@@ -470,10 +473,17 @@ def main():
     # create submission base data directory
     create_dir_as_python_package(SUBMISSION_DATA_BASE_DIR)
 
-    channel.queue_bind(exchange='evalai_submissions', queue='submission_task_queue', routing_key='submission.*.*')
-    channel.basic_consume(process_submission_callback, queue='submission_task_queue')
-
-    channel.queue_bind(exchange='evalai_submissions', queue=add_challenge_queue_name, routing_key='challenge.add.*')
+    channel.queue_bind(
+        exchange=RABBITMQ_PARAMETERS.get('EXCHANGE_NAME', None),
+        queue=RABBITMQ_PARAMETERS.get('SUBMISSION_QUEUE', None).get('NAME', None),
+        routing_key='submission.*.*')
+    channel.basic_consume(
+        process_submission_callback,
+        queue=RABBITMQ_PARAMETERS.get('SUBMISSION_QUEUE', None).get('NAME', None))
+
+    channel.queue_bind(
+        exchange=RABBITMQ_PARAMETERS.get('EXCHANGE_NAME', None),
+        queue=add_challenge_queue_name, routing_key='challenge.add.*')
```

Comment:
We use `.get` generally when we are not sure that the key will exist in dictionary or not. If we see the case here, this is something which is totally in our hand, so we can safely use `RABBITMQ_PARAMTERS['SUBMISSION_QUEUE']`